webapp/packages/core-cli/package.json

```diff
@@ -44,7 +44,7 @@
     "@cloudbeaver/tsconfig": "workspace:*",
     "@swc/core": "^1",
     "@types/js-yaml": "^4",
-    "@types/node": "^20",
+    "@types/node": "^22",
```

Comment:
After this update, you probably want to update "engines" to `"node": ">=22.0.0"`
Or you should keep the  ^20 version (to avoid use of not supported features)

In case we want to update to node >=22, then we need to ensure that it's updated in documentation and the devops team updates pipelines in Jenkins, also, maybe we will need to update GitHub Actions